src/platform/utilities/tests/environment/stagingDomains.unit.spec.js

```diff
@@ -1,15 +1,9 @@
 import { expect } from 'chai';
 
 import { replaceDomainsInData } from '../../environment/stagingDomains';
-import ENVIRONMENT from '../../../../site/constants/environments';
+import HOSTNAMES from '../../../../site/constants/hostnames';
 
-let currentEnv = '';
-if (global.__BUILDTYPE__ === ENVIRONMENT.VAGOVSTAGING) {
-  currentEnv = 'https://staging.va.gov';
-}
-if (global.__BUILDTYPE__ === ENVIRONMENT.VAGOVPROD) {
-  currentEnv = 'https://www.va.gov';
-}
+const currentEnv = `https://${HOSTNAMES[global.__BUILDTYPE__]}`;
```

Comment:
Ideally there would be a way to standardize accessing this, but the `buildtype` is available differently in Metalsmith/React/Mocha/E2E environments, so it's tough to do with making it overly complex.

Reply:
Feature flag api